### DISCUSSION TITLE
BT-767, BT-122-Lot, BT-133-Lot: applicability to groups of lots

### DISCUSSION THREAD

**Discussion by duncandewhurst** [OP-TED/eForms-SDK#134] [Q&A] [answered]
(Related to https://github.com/open-contracting/european-union-support/issues/71)

According to https://docs.ted.europa.eu/eforms/1.1.1/schema/all-in-one.html#auctionTermsSection, BT-767, BT-122 and BT-123 can apply to lots or to groups of lots:

> The use, for a given Lot, or Group of Lots, of an electronic auction system (BT-767) is specified with an indicator. When the value is "true", Description (BT-122) and URL (BT-123) may be specified.

However, in `fields.json`, these business terms only appear with a `-Lot` suffix, not a `-LotsGroup` suffix and the value of `xpathAbsolute` appears to restrict their use to lots only, e.g. for BT-122-Lot:

```
/*/cac:ProcurementProjectLot[cbc:ID/@schemeName='Lot']/cac:TenderingProcess/cac:AuctionTerms/cbc:Description
```
Can BT-767, BT-122 and BT-123 appear in the context of groups of lots?

**Comment by YvesJo** [comment, marked as answer]
Hi Duncan,
Thanks for pointing this out, the documentation will get updated. The text should read, instead:

> The use, for a given Lot, of an electronic auction system (BT-767) is specified with an indicator. When the value is "true", Description (BT-122) and URL (BT-123) may be specified.
